apps/examples/libcoap_server/Kconfig

```diff
@@ -0,0 +1,11 @@
+#
+# For a description of the syntax of this configuration file,
+# see misc/tools/kconfig-language.txt.
```

Comment:
Please change this line.
"# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt"